test/is.test.js

```diff
@@ -90,6 +90,25 @@ describe('is', () => {
     });
   });
 
+  describe('asyncFunction', () => {
+    it('should true', () => {
+      var asyncFun = async function() {
```

Comment:
@dead-horse Now it passed all tests. Maybe you can help rebasing before merging it into master. Thanks.